services/static-webserver/client/source/class/osparc/desktop/preferences/pages/GeneralPage.js

```diff
@@ -99,15 +93,15 @@ qx.Class.define("osparc.desktop.preferences.pages.GeneralPage", {
       });
       preferencesSettings.bind("creditsWarningThreshold", creditsWarningThresholdField, "value");
       creditsWarningThresholdField.addListener("changeValue", e => this.self().patchPreference("creditsWarningThreshold", creditsWarningThresholdField, e.getData()));
-      form.add(creditsWarningThresholdField, this.tr("Warning threshold"));
+      form.add(creditsWarningThresholdField, this.tr("Show Warning When Credits Below"));
 
       box.add(new qx.ui.form.renderer.Single(form));
 
       return box;
     },
     __createInactivitySetting: function() {
-      const box = this._createSectionBox(this.tr("Inactivity shutdown"));
-      const label = this._createHelpLabel(this.tr("Choose after how long should inactive studies be closed. A value of zero disables this function."));
+      const box = this._createSectionBox(this.tr("Automatic Shutdown of Idle Instances"));
+      const label = this._createHelpLabel(this.tr("Enter 0 to disable this function"), "text-13-italic");
```

Comment:
text-13-italic isn't valid and causes issues:
_Uncaught Error: Error in property font of class qx.ui.basic.Label in method $$setFontImpl with incoming value 'text-13-italic': Is invalid!_
See services/static-webserver/client/source/class/osparc/theme/Font.js for available fonts and definitions

Reply:
sorry my bad here. fixed it